.github/workflows/ci.yml

```diff
@@ -277,7 +283,6 @@ jobs:
       env:
         CC: /usr/bin/gcc-4.8
         CXX: /usr/bin/g++-4.8
-        GOOGLETEST_VERSION: "1.10.0"
```

Comment:
Why is this version requirement for GCC 4.8 removed?

Reply:
It is not removed, it was set in the build script. As I wrote above comments, "If GoogleTest version is not set on CI Pipeline, it was set as 1.10.0 by default. It could not be 1.12.1 version because GCC 4.8 oldish environment to support C++14."

Reply:
The logic is not intuitive, and convoluted in my opinion.

Instead of:

* setting a very old version by default in setup_cmake
* overriding with a new version in ci.yml in every places expect a few old builds

I suggest the opposite:

* Set by default the most up to date version in setup_cmake (1.13.0)
* Override the version in ci.yml for builds where we do want, on purpose, an older version (gcc 4.8, and the new C++ build to come in PR #2152)

Reply:
Marking as resolved, the version 1.10.0 is back in the CI script